repository/github_user.go

```diff
@@ -0,0 +1,96 @@
+//Package repository container the fuctions to handle githubusers model information
+package repository
+
+import (
+	"encoding/csv"
+	"io"
+	"log"
+	"lolidelgado/github-users/models"
+	"os"
+	"strconv"
+)
+
+type GithubUser struct {
+	githubUserCsvFileName string
+}
+
+type IGithubUserRepository interface {
+	FetchAll() ([]models.GithubUser, error)
+}
+
+const staticFilesPath = "static/"
```

Comment:
IMHO, this should be a field of your `NewGithubUser` struct, which needs to be sent from the main function, and also be a dynamic value.

Same reason as the fileName, if later you decide to change the location of the file, your codebase should remain the same; the only thing that would need to change is the value from config.